src/view/components/toolbar/ToolBar.tsx

```diff
@@ -150,22 +165,40 @@ class ToolBar extends React.Component<IProps, IToolbarState, any> {
         const component = content
             ? content.component
             : DEFAULT_MODAL_CONTENT.component;
+
         return (
             <div className="sensor_modal">
                 <div className="title_group">
                     <span className="title">
                         <FormattedMessage id={content.descriptionTitle} />
                         {content.tagInput}
                         {content.tagOutput}
+                        <IconButton
+                            onClick={this.onShowDescriptionClicked}
+                            iconProps={{ iconName: "Info" }}
+                            title="Info"
+                            ariaLabel="More Information Button"
+                            className="info-icon"
+                        />
                     </span>
                 </div>
                 <br />
-                <div className="description">
-                    <FormattedMessage id={content.descriptionText} />
-                </div>
+                {this.state.isDescriptionVisible && (
+                    <Callout
+                        className="description-callout"
+                        role="textbox"
+                        target=".info-icon"
+                        setInitialFocus={true}
+                        onDismiss={this.onDescriptionDismiss}
+                    >
+                        <div className="description">
+                            <FormattedMessage id={content.descriptionText} />
+                        </div>
+                    </Callout>
+                )}
                 <div className="try_area">
                     <br />
-                    <span className="description">
+                    <span className="try-it-description">
```

Comment:
This text is very difficult to read, it blends in the background 😭

Reply:
My bad, I meant this text:
![Annotation 2020-03-18 125448](https://user-images.githubusercontent.com/12337821/77001866-de1f9f00-6917-11ea-8f9c-cacff831636f.png)

Reply:
I noticed that too. I'll take a look into it.

Reply:
Updated, let me know what you think now.

Reply:
Much better, thank you!